api/buckets/client/client.go

```diff
@@ -345,11 +345,33 @@ func (c *Client) PullPathAccessRoles(ctx context.Context, key, pth string) (map[
 	return buckets.RolesFromPb(res.Roles)
 }
 
+func (c *Client) DefaultArchiveConfig(ctx context.Context, key string) (*pb.ArchiveConfig, error) {
+	res, err := c.c.DefaultArchiveConfig(ctx, &pb.DefaultArchiveConfigRequest{Key: key})
+	if err != nil {
+		return nil, err
+	}
+	return res.ArchiveConfig, nil
+}
+
+func (c *Client) SetDefaultArchiveConfig(ctx context.Context, key string, config *pb.ArchiveConfig) error {
+	req := &pb.SetDefaultArchiveConfigRequest{
+		Key:           key,
+		ArchiveConfig: config,
+	}
+	_, err := c.c.SetDefaultArchiveConfig(ctx, req)
+	return err
+}
+
 // Archive creates a Filecoin bucket archive via Powergate.
-func (c *Client) Archive(ctx context.Context, key string) (*pb.ArchiveResponse, error) {
-	return c.c.Archive(ctx, &pb.ArchiveRequest{
+func (c *Client) Archive(ctx context.Context, key string, opts ...ArchiveOption) error {
```

Comment:
Added options arg to specify a custom `ArchiveConfig`. This is also reflected in the local Buckets API.